src/sidebar/components/ToggleIconButton.tsx

```diff
@@ -0,0 +1,24 @@
+import type { IconButtonProps } from '@hypothesis/frontend-shared';
+import { IconButton } from '@hypothesis/frontend-shared';
+import classnames from 'classnames';
+
+export type ToggleIconButtonProps = IconButtonProps;
+
+/**
+ * An IconButton which can be used as a toggle with pressed and unpressed states
```

Comment:
All `IconButton`s can be used as toggle buttons, the distinction here is that the pressed state is indicated by a background change, which I would emphasize here. Other toggle buttons like menu dropdowns on the share button on annotation cards have other cues so they don't need this.

Reply:
Right, perhaps "toggle" was not the right term. I was thinking on a button that can be "on" or "off", and implicitly indicates it visually.

Perhaps `PressableIconButton` would be more clear?